prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -510,17 +510,22 @@ class ActionHistory {
         orgReceiver: Receiver,
         organization: Organization,
         sendingAt: OffsetDateTime?,
-        countToPrint: Int
+        countToPrint: Int,
+        reportOptions: ReportFunction.Options
     ) {
         jsonGen.writeStartObject()
         // jsonGen.writeStringField("id", reportFile.reportId.toString())   // TMI?
         jsonGen.writeStringField("organization", organization.description)
         jsonGen.writeStringField("organization_id", orgReceiver.organizationName)
         jsonGen.writeStringField("service", orgReceiver.name)
-        jsonGen.writeStringField(
-            "sending_at",
-            if (sendingAt == null) "immediately" else "$sendingAt"
-        )
+
+        if(reportOptions == ReportFunction.Options.SkipSend) {
+            jsonGen.writeStringField("sending_at", "never - skipSend specified")
+        }
+        else {
+            jsonGen.writeStringField("sending_at", if (sendingAt == null) "immediately" else "$sendingAt")
+        }
+
```

Comment:
@pablocaviglia , try rewriting this using the Kotlin "when" statement - you'll find lots of examples of it in our code, plus stuff on the web.   Its like a "switch" statement but more cool and more powerful.   When you have three or more choices, that's a time to think a "when" statement might be appropriate.   Enjoy!

Reply:
fixed @jimduff-usds

Reply:
looks good - one minor thing - I think standard Kotlin is to use .isNull instead of == null
Are you using IntelliJ?  If so, it gives you nice tips/fixes like this.

Reply:
Uhmmm... I dont know that function, or extension function... neither the IDE 

![image](https://user-images.githubusercontent.com/71846466/121698427-5ede3400-caa4-11eb-81e9-2535af07ccf3.png)

Reply:
There should be an sendingAt.isNullOrEmpty and .isNullOrBlank

But never mind, let's just get this merged in.

Reply:
Ohh I see what are you saying, but it only applies to strings, not for OffsetDateTime like sendingAt variable

![image](https://user-images.githubusercontent.com/71846466/121699317-34d94180-caa5-11eb-834e-22198f88859d.png)